packages/plugins/plugin-automation/src/components/TriggerEditor/TriggerEditor.tsx

```diff
@@ -54,7 +54,7 @@ export const TriggerEditor = ({ space, trigger, onSave, onCancel }: TriggerEdito
 
         const handleOnValueChange = useCallback(
           (_type: any, dxnString: string) => {
-            const dxn = DXN.parse(dxnString);
+            const dxn = Type.DXN.parse(dxnString);
```

Comment:
is the expectation that DXN will always be accessed through `Type`? even for non-schema things?